src/qml/LinePolygon.qml

```diff
@@ -0,0 +1,43 @@
+import QtQuick 2.14
+import QtQuick.Shapes 1.14
+
+import Theme 1.0
+import org.qfield 1.0
+import org.qgis 1.0
+
+LinePolygonShape {
+  id: linePolygonShape
+
+  onPolylinesChanged: {
+    var pathElements = []
```

Comment:
```suggestion
    const pathElements = []
```

We don't change the reference of that var, so it should be `const`.

Reply:
Do we add semicolons or not? Personally I am fan of semicolons, the codebase is mixed. What do we do with new code like this? No?